packages/plugins/experimental/plugin-map/src/hooks/useStabilizedObject.ts

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { useState, useEffect } from 'react';
+
+export const useStabilizedObject = <T>(obj: T, customEqual: (oldObject: T, newObject: T) => boolean) => {
```

Comment:
when making a generic util like this try to include a jsdoc about the intended use case/what it's for

Reply:
Good idea, done.